src/App.js

```diff
@@ -1,24 +1,27 @@
-import logo from './logo.svg';
 import './App.css';
+import { Routes, Route } from 'react-router-dom'
+import { Header } from './components/Header';
+import { Nav } from './components/Nav';
+import { User } from './components/User';
+import { ReviewList } from './components/ReviewList/Review-list';
+import { Review } from './components/Review/Review';
+
+import {useState} from 'react'
 
 
 function App() {
+  const [CurrUser, setCurrUser] = useState("Bobby")
```

Comment:
States shouldn't be capitalised, they're regular variables so lower camel case please. 

FYI if you're hard coding a user pick one that exists in the api, Bobby ain't in there 😁